README.md

```diff
@@ -67,6 +81,22 @@ All the links are monitored and tested with [awesome_bot](https://github.com/dkh
 	- [Russian](#russian)
 
 
+Icons and tags:
+
+- Ancient: ![ancient]
+- Commercial: ![$]
+- Crush: ![heart]
+- Supporting: ![support]
+- Up and coming: ![upcoming]
+- Web Based: ![web]
+- Beta: ![beta]
+- Video: ![video]
+- Open Source: ![oss]
+- CLI/Terminal: ![cli]
+- Windows: ![windows]
```

Comment:
If we add an icon for supports OSX or Linux it will get very cluttered, But maybe we use it in reverse i.e. Windows, Linux, OSX only tools